components/DateTimePicker/DatePicker.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+
+import { useState } from 'react';
+import { format } from 'date-fns';
+
+import { Calendar } from '@/components/ui/calendar';
+import InputField from '@/components/InputField/InputField';
+
+/**
+ * DatePicker 컴포넌트는 달력을 표시하고 선택한 날짜를 반환합니다.
+ * @param {string} width - optional, DatePicker 컴포넌트의 너비를 설정합니다. (기본 값은 w-full입니다.)
+ * @returns {JSX.Element} DatePicker 컴포넌트를 반환합니다.
+ */
+
+export default function DatePicker({ width = 'full' }: { width?: string }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [date, setdate] = useState<Date | undefined>(undefined);
+  const formattedDate = date ? format(date, 'yyyy년 mm월 dd일') : '';
+
+  return (
+    <>
+      <div
+        className={`${`${width !== 'full' ? `w-pr-${width}` : 'w-full'}`} flex flex-col gap-pr-8`}
+      >
```

Comment:
tailwind에서 동적 유틸 클래스 생성은 권장하지 않는 방식 입니다 ! [참고](https://tailwindcss.com/docs/detecting-classes-in-source-files)
width만 props로 받아서 완성형 유틸 클래스를 리턴하는 유틸 함수를 이용해주세요!

```suggestion
        className={`${width ? widthStyledSliceWPr(width) : 'w-full'} flex flex-col gap-pr-8`}
```

Reply:
다 수정되었습니다~

Reply:
어제 팀미팅 시간에 말씀하셨던 input에 value가 있을 때 텍스트가 흰색으로 바뀌는 부분은 inputfield에서 수정이 되어야 할 거 같습니당

Reply:
확인 부탁드립니당

Reply:
여기 부분 전체 수정이 안되었습니당